service/controller/v24/controllercontext/status.go

```diff
@@ -1,25 +1,22 @@
 package controllercontext
 
-type Status struct {
-	// Cluster carries information between cluster controller resource.
-	Cluster Cluster
-	// Drainer carries information between drainer controller resource.
-	Drainer Drainer
+type ContextStatus struct {
+	ControlPlane  ContextStatusControlPlane
+	TenantCluster ContextStatusTenantCluster
 }
 
-type Cluster struct {
-	AWSAccount             ClusterAWSAccount
-	ASG                    ClusterASG
+type ContextStatusControlPlane struct {
+	AWSAccountID string
+}
+
+type ContextStatusTenantCluster struct {
+	AWSAccountID           string
+	TCCP                   ContextStatusTenantClusterTCCP
```

Comment:
For node pools we need to start differentiating between different stacks. Here is the stack reflected with the resource name. Later we add TCDP which will be the Tenant Cluster Data Plane in which the node pool stacks are managed.